modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -57,6 +57,13 @@ properties:
               Introduced in Sync Gateway 2.0, this property can be used to disable Sync Gateway's handling of conflicts.
 
               Setting to `false` will cause Sync Gateway to reject any attempt to write conflicting revisions (returning a `409` HTTP status code). It will be up to the client to resolve the conflict. Restarting Sync Gateway with this property enabled will not automatically result in disk space savings (compaction on a document won't occur until a document is updated).
+
+              In a deployment where replication between Sync Gateway clusters is used (i.e [sg-replicate](running-replications.html)), the `allow_conflicts` property is only supported in the following scenarios.
+
+              - *One-way replication:* replication between Sync Gateway clusters is configured to be only pull or push.
+              - *Single writer:* only 1 of the Sync Gateway clusters is processing incoming write operations.
+
+              Consequently, this property is not supported with bi-directional replication between Sync Gateway clusters or when both clusters are accepting write operations.
```

Comment:
As might be expected by the name (allow_conflicts), this is only issue if conflicting updates to the document are being made in both clusters.  

It might be clearer to state that sg-replicate doesn't support automatic conflict resolution.  When running w/ allow_conflicts=false, cross-cluster document conflicts will result in that document no longer being replicated.  To avoid this, users that want to use allow_conflicts=false need to ensure that concurrent conflicting updates are not made to a given document.

This differs from the above, in that it's fine for writes to occur in both clusters, as long as the same document isn't being written concurrently (i.e. within the replication window) to both clusters.

Reply:
Understood, will update.